runtime/src/main/scala/akka/grpc/internal/NettyClientUtils.scala

```diff
@@ -57,6 +65,7 @@ object NettyClientUtils {
           ChannelUtils.monitorChannel(promise, channel, settings.connectionAttempts)
           Future.successful(channel)
         } else {
+          promise.tryFailure(new NoTargetException("No targets returned for name: " + settings.serviceName))
```

Comment:
Perhaps return this exception as well?

Reply:
I would be fine with it, but this would break other people's code if they depend on the exception, are you fine with this?

Reply:
Yes, I don't think it is likely people were depending on the specific exception, this is a clear improvement and we're 'allowed' to make such changes since we're still in the 0.x.y phase.

Reply:
Alright, changed